bosch_thermostat_client/bosch_cli.py

```diff
@@ -76,7 +76,9 @@ async def _scan(gateway, smallscan, output, stdout):
 
 async def _runquery(gateway, path):
     _LOGGER.debug("Trying to connect to gateway.")
-    result = await gateway.raw_query(path)
+    result = []
+    for p in path:
+        result.append(await gateway.raw_query(p))
```

Comment:
I'd add `await asyncio.sleep(0.3)` just to be sure that one query after another won't lock you from API.

Reply:
I didn't found a similar sleep in helper.py#deep_into thats why I didn't add smth like that but maybe I missed something. Anyway - you are far more experienced with that code and those devices :smile:. I added the sleep.